### ISSUE TITLE
Nexus tree file with internal comments

### ISSUE THREAD

**Issue by josephwb** [FePhyFoFum/phyx#98]
Make the tree reader fail. Similar to #80. Example:
```
#NEXUS 

Begin trees;  [Treefile saved Thu Feb 21 14:21:24 2019]
[!
>Consensus of 19848 trees
>
>Source of trees from which consensus(es) calculated...
>Heuristic search settings:
>  Optimality criterion = parsimony
>    Character-status summary:
>      Of 146 total characters:
>        12 characters are of type 'ord' (Wagner)
>        134 characters are of type 'unord'
>        All characters have equal weight
>        3 characters are parsimony-uninformative
>        Number of parsimony-informative characters = 143
>    Gaps are treated as "missing"
>    Multistate taxa interpreted as uncertainty
>  Starting tree(s) obtained via stepwise addition
>    Addition sequence: random
>    Number of replicates = 10000
>    Starting seed = generated automatically
>    Number of trees held at each step = 1
>  Branch-swapping algorithm: tree-bisection-reconnection (TBR) with reconnection limit = 8
>    Steepest descent option not in effect
>  Initial 'Maxtrees' setting = 100 (will be auto-increased by 100)
>  Branches collapsed (creating polytomies) if maximum branch length is zero
>  'MulTrees' option in effect
>  No topological constraints in effect
>  Trees are unrooted
>
>Heuristic search completed
>  Total number of rearrangements tried = 4.8079e+10
>  Score of best tree(s) found = 658
>  Number of trees retained = 19848
>  Time used = 02:10:07 (CPU time = 02:10:07.5)
]
	tree Strict = [&U] (Eudromia_elegans,Dendrocygna_bicolor,(((((Psittacus_erithacus,Cacatua_su
lphurea,Nestor_meridionalis),(Cyrilavis_colburnorum,Pseudasturides_macrocephalus)),((Anneavis_anneae
,Tsidiiyazhi_abini,Sandcoleus_copiosus),((Oligocolius_psittacocephalon,(Colius_striatus,Urocolius_in
dicus)),Masillacolius_brevidactylus,Chascacocolius_oscitans,Selmes_absurdipes)),((Leptosomus_discolo
r,Plesiocathartes_wyomingensis),Plesiocathartes_kelleri),(Paracoracias_occidentalis,Eocoracias_brach
yptera,(Coracias_caudatus,Atelornis_pittoides),(Megaceryle_alcyon,Dacelo_novaeguineae),Merops_ornatu
s,Momotus_momota,(Palaeotodus_itardiensis,Todus_subulatus)),Harpactes_erythrocephalus,Trogon_melanur
us,Primotrogon_wintersteini,Messelirrisor_halcyrostris,(Upupa_epops,Tockus_camurus),((Rupelramphasto
ides_knopfi,Dryocopus_pileatus),Galbula_albirostris),(((Tyrannus_tyrannus,Pitta_erythrogaster),(Acan
thisitta_chloris,((Corvus_brachyrhynchos,Gracula_religiosa),Spinus_pinus)),Wieslochia_weissi),Zygoda
ctylus_luberonensis,Eozygodactylus_americanus,Primozygodactylus_quintus),(Eofringillirostrum_boudrea
uxi,Eofringillirostrum_parvulus),Morsoravis_sedilis,Morsoravis_sp.,Pumiliornis_tessellatus,Psittacop
es_lepidus,Eocuculus_cherpinae),(((Falco_peregrinus,Buteo_jamaicensis),Sagittarius_serpentarius),Cat
hartes_aurea),(Tyto_alba,Bubo_virginianus)),Cariama_cristata));
end;
```
Such trees are produced by PAUP and Seq-Gen.

**Comment by josephwb** [comment]
Should be fixed with c2739c64411a85ec5af47b27ff6de65ebcd6c9c6. However, there was a weird merge conflict that I will check.

**Comment by josephwb** [comment]
Ok, merge conflict was with myself (multiple computers) so everything is kewl.
